boards/arm/adafruit_feather_m4_express/adafruit_feather_m4_express.yaml

```diff
@@ -0,0 +1,20 @@
+identifier: adafruit_itsybitsy_m4_express
```

Comment:
The buildkite check is failing because this needs to be changed to the new board name